Jenkinsfile

```diff
@@ -1053,10 +1053,11 @@ pipeline {
     } // stages
     post {
         always {
-          job_status_update('final_status')
-          job_status_write()
```

Comment:
You probably don't want to be removing these do you?  Maybe needs a master merge?

Reply:
Must have been a bad merge somewhere.   They should be back now.